Update stlSerializer.ts

aWeirdo opened the pull request:
Fix wrong orientation, flipped faces and bloated binary export.

-- Original mesh
![head_original](https://user-images.githubusercontent.com/4191201/95802127-e056aa00-0cfc-11eb-81ea-7c8ede4d31fe.png)

-- After current export
![head_export_now](https://user-images.githubusercontent.com/4191201/95802141-e77db800-0cfc-11eb-8ce9-a3354b255178.png)

-- After fixed export
![head_export_fixed](https://user-images.githubusercontent.com/4191201/95802144-e8aee500-0cfc-11eb-9ae3-710feb1c8a9c.png)

RaananW commented on serializers/src/stl/stlSerializer.ts:
```diff
@@ -12,23 +12,23 @@ export class STLExport {
             let p1p2 = v[0].subtract(v[1]);
             let p3p2 = v[2].subtract(v[1]);
-            let n = (Vector3.Cross(p1p2, p3p2)).normalize();
+            let n = (Vector3.Cross(p1p2, p3p2)).normalize().negateInPlace();
```
Just wondering about the normal negation - why is this needed?

Popov72 replied:
It must be because the vertices are now x/z/y instead of x/y/z.

But it may be easier to simply do: `let n = (Vector3.Cross(p3p2, p1p2)).normalize()`

aWeirdo commented:
Last minute changes,
Removed n vector3 negating & switched the input values as suggested.
Updated default fileName to match binary mesh name (which is always 'stlmesh')
Updated ASCII mesh name to match binary mesh name for consistency
And removed fileName check in download code as it is defaulted at beginning.
